tests/parsec-cli-tests.sh

```diff
@@ -286,6 +298,7 @@ Usage: $0 [parameter]
     -N:   Test only the provider with N ID
     --no-oaep: Do not test RSA-OAEP(SHA256) encryption/decryption operations
     --no-v1.5: Do not test RSA-PKCS#1-v1.5 encryption/decryption operations
+    --rsa-key-size: Perform all RSA operations with a fixed key length
```

Comment:
--rsa-key-size: Perform all RSA operations with a fixed key length with the specified key length?

Reply:
Sorry I didnt get it. Do you want me to reword it to 
`--rsa-key-size: Perform all RSA operations with the specified key length`?

Reply:
yes, if you agree